app/Models/Router.php

```diff
@@ -48,11 +48,17 @@ public function getFailStartDate()
 
     public function sendWarning()
     {
-        if ($this->failed_for == self::WARNING_THRESHOLD) {
+        if ($this->failed_for == self::WARNING_THRESHOLD && env('APP_DEBUG') == false) {
```

Comment:
I meant do not increment the failed_for attribute if the app is in debug mode. I'm afraid  this way we still get emails after changing back to prod.